app/models/course.rb

```diff
@@ -78,4 +78,19 @@ def open_for_applications?
   def has_vacancies?
     site_statuses.with_vacancies.any?
   end
+
+  def study_mode_description
+    study_mode.to_s.tr("_", " ")
```

Comment:
Minor suggestion: part of me would prefer a more explicit `case` here in which we opt into the strings that we want to translate, rather than automating it with `tr`, and set a `default` for unexpected cases. But this is more pragmatic and should work just fine.